client-linuxapp/src/serviceinfo.rs

```diff
@@ -52,6 +52,22 @@ fn set_perm_mode(path: &Path, mode: u32) -> Result<()> {
     Ok(())
 }
 
+fn create_user(user: &str) -> Result<()> {
+    let user_info = passwd::Passwd::from_name(user);
+    if user_info.is_none() {
+        log::info!("Createing user: {}", user);
+        Command::new("useradd")
+            .arg(format!("-m {}", user))
```

Comment:
This would lead to a single command "-m {username}", so it would end up getting called like: `useradd "-m someusername"`, which isn't the way things are expected.
You should do a `.arg("-m").arg(user)`, to make sure it becomes to separate arguments.